wordpress-updatable/Makefile

```diff
@@ -0,0 +1,136 @@
+APP_ID ?= example/wordpress-updatable
+
+# Track should be the major.minor of RELEASE.
+TRACK ?= 0.1
+
+# Check directory versions/ for all available release values.
+RELEASE ?= 0.1.1
```

Comment:
Please consider using one version of manifest (without the `versions` directory), and providing dynamically an app version to the `application.yaml.template`.

Example: https://github.com/GoogleCloudPlatform/click-to-deploy/blob/5ee38280f34b3eb9ca515a053cd90c64dfb5098a/k8s/wordpress/chart/wordpress/templates/application.yaml.template#L16

Reply:
I thought about doing it, but I think such automation should be left to the partner, depending on how they would want to do it. 

In this particular case, we want to be clear that the two versions are technically separate. Adding more automation/magic into the example hides the important pieces.